Frameworks/Foundation/NSKeyValueCoding.mm

```diff
@@ -674,7 +674,7 @@ - (void)performSelector:(SEL)selector onThread:(NSThread*)thread withObject:(id)
     } else {
         if (runloop == nil) {
             [NSException raise:NSInvalidArgumentException
-                        format:@"Can't perform selector %2: Thread %@ has no runloop", NSStringFromSelector(_cmd), thread];
+                        format:@"Can't perform selector %@: Thread %@ has no runloop", NSStringFromSelector(_cmd), thread];
```

Comment:
nit: Some men see things as they are and ask why.  I dreams things that never were and ask why not.